lib/teiserver/telemetry/schemas/simple_server_event_type.ex

```diff
@@ -11,6 +11,8 @@ defmodule Teiserver.Telemetry.SimpleServerEventType do
   """
   @spec changeset(Map.t(), Map.t()) :: Ecto.Changeset.t()
   def changeset(struct, params \\ %{}) do
+    IO.inspect(struct, label: "Struct:")
+    IO.inspect(params, label: "Params:")
```

Comment:
I just noticed that, you probably don't want to commit these debug lines.

Reply:
I saw them yesterday as well and removed them locally already